crates/broker/src/aggregator.rs

```diff
@@ -406,6 +406,56 @@ impl AggregatorService {
         Ok(false)
     }
 
+    /// Filter out expired orders and mark them as failed
+    async fn filter_expired_orders(
+        &self,
+        orders: Vec<AggregationOrder>,
+        current_time: u64,
+    ) -> Result<Vec<AggregationOrder>, AggregatorErr> {
+        let mut valid_orders = Vec::with_capacity(orders.len());
+
+        for order in orders {
+            if order.expiration < current_time {
+                tracing::warn!(
+                    "Order {} has expired during aggregation, marking as failed",
+                    order.order_id
+                );
+                if let Err(err) =
```

Comment:
Definitely hesitant to add more alerts, but I do worry that now when this happens the problem is essentially swallowed, and we won't know. And its pretty bad if this happens since you get slashed.

Thoughts on something like an alert where if it happens a few times an hour we get a SEV2?

Reply:
very reasonable, I set it to a low threshold (2) given this expiry timing is basically an edge case, so if it happens more than once, likely something wrong